packages/get-starknet/README.md

```diff
@@ -0,0 +1,23 @@
+# MetaMask Get Starknet
+
+This repository contains a node module that makes it easy to integrate the Starknet Snap through the interface defined by get-starknet-core.
+
+
+## how to build
+Execute the following cmd to build the project in [module federation]([https://webpack.js.org/concepts/module-federation/] ) standard
+```bash
+yarn build:fed
+```
+
+
+## How to use
+
+```javascript
+const walletInstance = new MetaMaskSnapWallet(
```

Comment:
Can you also include the statements for importing the `MetaMaskSnapWallet()` object in this example ?

Reply:
agree

Reply:
but i will refer to get-starknet v3 user guide, as our interface is implemented with get-starknet

Reply:
the package is for get-starknet, but no one else, i think we dont need a readme